packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -197,9 +197,10 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
       classNames.footer,
       {
         display: 'flex',
+        alignItems: 'center',
+        color: palette.white,
         selectors: {
-          // TODO: global class name usage should be converted to a button styles function once Button supports JS styling
-          [`.${classNames.button}:not(:first-child)`]: {
+          '> *:not(:first-child)': {
```

Comment:
What is driving this change?

Reply:
I changed the selector to target all direct children of `footer` to avoid having to repeat `marginLeft: '20px'` for the buttons and for the custom footer text. 

Also, the only children of the footer container div are the buttons and now the custom footer text that I added. I don't think it's necessary to target the button classes specifically. Again let me know if my thinking is correct here, I can revert if appropriate